{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/utils/views.py

```diff
@@ -7,3 +18,36 @@ def get_template_names(self):
         template_name = self.kwargs.get('slug', '_base')
 
         return ['frontend/{}.html'.format(template_name)]
+
+
+class FrontendEditView(View):
+    def post(self, request):
+        if request.user.is_superuser:
+            data = json.loads(request.body.decode('utf-8'))
+            app_name = data.get('app', False)
+            model_name = data.get('model', False)
+            if app_name and model_name:
+                model = apps.get_model(app_label=app_name, model_name=model_name)
+                pk = data.get('pk', False)
```

Comment:
Same as before: is there any time this will be empty under reasonable circumstances? Let this fail loudly if not. (And it simplifies the `if` branches below.